python/README.rst

```diff
@@ -6,34 +6,57 @@ This package provides bindings to the deltachat-core_ C-library
 which provides imap/smtp/crypto handling as well as chat/group/messages
 handling to Android, Desktop and IO user interfaces.
 
-Install
-=======
+Installing pre-built packages (linux-only)
+==========================================
 
-You may also want to build a wheel using docker instead of manually
-building deltachat-core.  See below for this.
+If you have a linux system you, may install
+``deltachat`` and all of its dependencies like this:
 
-1. First you need to `install the delta-core C-library
-   <https://github.com/deltachat/deltachat-core/blob/master/README.md>`_.
-
-2. `Install virtualenv <https://virtualenv.pypa.io/en/stable/installation/>`_
-   if you don't have it, then create and use a fresh clean python environment::
+1. `Install virtualenv <https://virtualenv.pypa.io/en/stable/installation/>`_,
+   then create a fresh python environment and activate it in your shell::
 
         virtualenv -p python3 venv
         source venv/bin/activate
 
-   Afterwards invoking ``python`` or ``pip install`` will only modify files
-   in your ``venv`` directory.
+   Afterwards, invoking ``python`` or ``pip install`` will only
+   modify files in your ``venv`` directory and leave your system installation
+   alone.
 
-3. Install the bindings with pip::
+2. Install the wheel for linux::
 
         pip install deltachat
 
-   Afterwards you should be able to successfully import the bindings::
+    Verify it worked by typing::
 
         python -c "import deltachat"
 
-You may now look at `examples <https://py.delta.chat/examples.html>`_.
 
+Installing a wheel from a github branch
+---------------------------------------
+
+For Linux, we automatically build wheels for all PRs
+and push them to a devpi/pypi index which case be used
+with ``pip``.  For example, to install the latest "master" built::
+
+    pip install -i https://m.devpi.net/dc/master deltachat
+
+
+Installing bindings from source
+===============================
+
+If you can't use "binary" method above then you will need
+to `install the delta-core C-library <https://github.com/deltachat/deltachat-core/blob/master/README.md>`_ before you invoke the ``pip install deltachat`` command above.
```

Comment:
`pip install --no-binary deltachat` to avoid using the wheel and compile against the system libdeltachat.so.  This is useful in case pypi has an outdated libdeltachat.so because we forgot/got hit by a bus.